crates/sui-source-validation-service/tests/tests.rs

```diff
@@ -183,6 +191,7 @@ fn test_clone_command() -> anyhow::Result<()> {
         repository: "https://github.com/user/repo".into(),
         branch: "main".into(),
         paths: vec!["a".into(), "b".into()],
+        network: Some(Network::Localnet),
     };
 
     let command = CloneCommand::new(&source, PathBuf::from("/tmp").as_path())?;
```

Comment:
Do we actually use `/tmp` for anything in this case? If so, we should use `tempfile` for this instead, because `/tmp` is not going to be available on all systems. (I suspect for this test, we don't, but my eye was just drawn to it here).

Reply:
We don't use it for anything, it's just to test the command construction, I'll rename it to something like `/foo` in follow up so this isn't flagged if someone thinks it's actually using /tmp